openpower/package/petitboot/66-add-sg-module.rules

```diff
@@ -0,0 +1,2 @@
+# load modules to scsi disks, if they aren't in kernel
+SUBSYSTEM=="scsi_device", ACTION=="add", RUN+="/sbin/modprobe sg"
```

Comment:
this will try to insert the module for every SCSI device; we probably want something along the lines of the distros' udev rules that do this:

```SUBSYSTEM=="scsi", ENV{DEVTYPE}=="scsi_device", TEST!="[module/sg]", RUN+="/sbin/modprobe sg"```

(and can we use the udev-builtin kmod here?)